contentgrid-spring-integration-events/src/main/java/com/contentgrid/spring/integration/events/EntityChangeEventTransformer.java

```diff
@@ -19,7 +20,7 @@ public class EntityChangeEventTransformer extends AbstractPayloadTransformer<Ent
     @Override
     protected ChangeEventPayload transformPayload(EntityChangeEvent changeEvent) {
         return new ChangeEventPayload(
-                changeEvent.getTrigger(),
+                changeEvent.getTrigger().toString().toLowerCase(Locale.ROOT),
```

Comment:
This should either also use `.name()`, like in `EntityChangeEventPublisher`, or both should be using `toString()`

I prefer using `.name()`, as it is more obvious to me that it's the enum name. For `toString()`, I always have to remember that enum does not use the default `Object#toString()` implementation.